frontend/src/v5/ui/controls/circleButton/circleButton.component.tsx

```diff
@@ -15,18 +15,26 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import React from 'react';
+import React, { Dispatch } from 'react';
 
 import { StyledFab } from './circleButton.styles';
 
 interface ICircleButton {
 	size?: 'large' | 'medium' | 'small';
 	variant?: 'main' | 'contrast';
+	onClick: Dispatch<void>;
 	disabled?: boolean;
+	className?: string;
 }
 
-export const CircleButton: React.FC<ICircleButton> = ({ size = 'large', variant = 'main', children, ...props }) => (
-	<StyledFab size={size} variant={variant} {...props}>
+export const CircleButton: React.FC<ICircleButton> = ({
```

Comment:
Why are you making changes to a component that You didn't even use for this task? This approach can create unnecessary conflicts. Remember that you are not the only one working on this code.

Reply:
You're right. It's a leftover and it shouldn't be in the PR.

Reply:
Do we agree that these changes should be removed from this PR?

Reply:
Sure. I've missed it.